elements/pfe-accordion/src/pfe-accordion-panel.scss

```diff
@@ -48,3 +48,9 @@ $LOCAL: accordion;
     @include pfe-panel-variables($theme: #{$context});
   }
 }
+
+:host([pfe-disclosure="true"]) {
+  .container {
+    padding-left: calc((#{pfe-var(container-spacer)} * 1.5) - 4px);
```

Comment:
we have --pfe-accordion--triggerPadding, maybe we should mark this as a todo for the future but the trigger padding and the container padding should probably be related

Reply:
Just trying to figure out a simple way to do this. This is the padding currently:
```
**disclosure padding**
* triggerPadding: 16px 24px 16px 47px; _set in @mixin pfe-trigger_
* panel padding: 0 48px 16px 20px; _set in @mixin pfe-panel-container_

**accordion padding**
* triggerPadding: 16px 50px 16px 24px;
* panel padding: 0 48px 16px 24px;
```

Reply:
@starryeyez024 I set a `PaddingBase` variable that is applied to both the `Trigger--padding` and the `PanelContainer--padding` variables. If the `PaddingBase` value (currently equal to `container-spacer` changes it will change the trigger and container padding.

Reply:
thanks!